### PR TITLE
[Do Not Merge] Add Windows Nano Server outerloop job to Jenkins.

### PR DESCRIPTION
**Pull request by Priya91**
cc @mmitche @joshfree @ellismg 

This change requires Unpacker.exe to work for windows, and also requires this dotnet/dotnet-ci#196
[UPDATE] Both of these issues are fixed.

### CODE REVIEW

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -104,10 +105,87 @@ branchList.each { branchName ->
...
+            		stringParam('COREFX_BUILD', '', 'Build number to use for copying binaries for nano server bld.')
+            	}
+            }
+
+            // Set the affinity.  All of these run on Windows Nano currently.
+            Utilities.setMachineAffinity(newTestJob, os, 'latest-or-auto')
+            // Set up standard options.
+            Utilities.standardJobSetup(newTestJob, project, isPR, getFullBranchName(branchName))
+
+            def newJob = job(getJobName(Utilities.getFullJobName(project, newJobName, isPR), branchName)) {
```
Comment (mmitche) [inline netci.groovy]: I think this needs to be buildFlowJob

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
@dotnet-bot test ci please

**Comment by Priya91** [comment]
Rebased all commits,

@dotnet-bot test ci please

**Pull request merged**